utils/build_ds_container.py

```diff
@@ -42,6 +42,11 @@
         '--product and --debug flags.'),
     action='store_true',
     default=False)
+parser.add_argument(
+    '-i', '--push-content-image',
+    help=(
+        'Do not build any content. Create profile bundles from the referenced k8s content image'),
```

Comment:
Perhaps this means we need to separate image push functionality into a separate tool, since we're invoking a script with `build` but explicitly telling it to not build.

Reply:
Yes, it felt weird doing this but I wanted reuse code we already had.

Reply:
Yep - that makes sense. If we break this out, then I'd say we should get the `build_ds_container.py` script to use the image push functionality, so we don't repeat ourselves.